CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ### 1.18.9 / 2020-08-04
+* Change windows 2012r2 VM to work around issues
+* Use same bundler instance to work with BEAKER_GEM_OPTIONS
```

Comment:
Why use the same bundler instance instead of persisting `BEAKER_GEM_OPTIONS` into the clean bundle?

Reply:
@op-ct Out of curiosity, why did we have the clean bundle in the first place?

Reply:
:thinking: